app.go

```diff
@@ -33,6 +35,57 @@ type appImpl struct {
 	pluginMngr PluginManager
 	config     Config
 	mFS        []ManagedFS
+	processors map[string]ValueProcessor
+}
+
+// ProcessorCallback is a function signature used as a callback in processors.
+type ProcessorCallback func(value interface{}, options map[string]interface{}) (interface{}, error)
```

Comment:
These all must be defined inside internal package as other similar types, because it must be available inside the launchr also.